aoc/src/main/scala/aoc2022/exercise05.scala

```diff
@@ -0,0 +1,84 @@
+package aoc2022
+
+import aoc2022.standardAoc.readInput
+import scala.collection.mutable.Map
+import scala.io.Source
+
+
+
+object exercise05 extends App:
+  // Read input
+  var input = readInput("exercise05")
+    .mkString("\n")
+    .split("\n\n")
+    .toList
+
+  // Get the action part
+  val actions = readInput("exercise05")
+    .mkString("\n")
+    .split("\n\n")
+    .toList(1).split("\n").toList
+
+  // Get the stack part
+  var stacks = input
+    .toList(0)
+    .replace("    ", "[XX]")
```

Comment:
nicely solved - and very concise - still fww - have a look at [this](https://github.com/nmcb/aoc2022/blob/main/src/main/scala/Day05.scala#L16-L51) parser code bc im thinking it is better readable